src-electron/generator/helper-zcl.js

```diff
@@ -528,23 +529,40 @@ async function zcl_commands_with_cluster_info(options) {
 async function zcl_commands_with_arguments(options) {
   let sortBy = options.hash.sortBy
   let packageIds = await templateUtil.ensureZclPackageIds(this)
+  let packages = await queryPackage.getPackagesByPackageIds(
+    this.global.db,
+    packageIds
+  )
 
   let cmds = await Promise.all(
-    packageIds.map(async (packageId) => {
+    packages.map(async (pkg) => {
       let cmdsPerPackageId = await queryCommand.selectAllCommandsWithArguments(
         this.global.db,
-        packageId
+        pkg.id
       )
       if ('signature' == sortBy) {
-        for (const cmd of cmdsPerPackageId) {
-          let sig = await zclUtil.createCommandSignature(
-            this.global.db,
-            packageId,
-            cmd
-          )
-          cmd.signature = sig.signature
-          cmd.isSignatureSimple = sig.isSimple
+        if (pkg.type === dbEnum.packageType.zclXmlStandalone) {
+          for (const cmd of cmdsPerPackageId) {
+            let sig = await zclUtil.createCommandSignature(
+              this.global.db,
+              packageIds,
```

Comment:
Remove code redundancy in the if and else statements because this is the only change. What about?

pkg.type === dbEnum.packageType.zclXmlStandalone ? packageIds : pkg.id